src/ORM/Behavior/TreeBehavior.php

```diff
@@ -75,6 +75,7 @@ class TreeBehavior extends Behavior
         'scope' => null,
         'level' => null,
         'recoverOrder' => null,
+        'useOrmDelete' => false,
```

Comment:
I don't know why, but the name feels kinda odd... in associations we have `cascadeCallbacks` that configure basically the same, so for sake of consistency it could make sense to name the option something with `callbacks` too? 🤔

Reply:
+1 to what ndm2 said.

Reply:
I like that idea even better

Reply:
Might be a little misunderstanding, I didn't mean to say to use the same name, though, thinking about, to some extent it is a cascading delete, given that a tree branch is naturally a cascading structure.